sulong/projects/com.oracle.truffle.llvm.parser.factories/src/com/oracle/truffle/llvm/parser/factories/LinuxAArch64PlatformCapability.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates.
+ * Copyright (c) 2020, Arm Limited. All rights reserved.
```

Comment:
Please drop the "All rights reserved." In the BSD header this is superfluous since its already there just 2 lines below.

Reply:
Done.